### DISCUSSION TITLE
MouseOver behaviour

### DISCUSSION THREAD

**Discussion by axtens** [unoplatform/uno#14187] [Q&A] [answered]
It has finally dawned on me that MouseOver behaviour is a feature not a bug; that a control has behaviours based on whether the mouse pointer is over it or not.

Okay, so supposing I don't want that behaviour in my RichEditBox, how do I do that? I want my RichEditBoxes to have #F9F8BD as the standard background colour. I don't want that changing when the mouse button hovers over it. What changes must I make to my XAML?

-Bruce

**Comment by jeromelaban** [comment]
Most of the time, overriding the `RichEditBox` will do. Have you tried that?

**Reply by axtens** [reply]
I don't know how to do that.

**Comment by axtens** [comment, marked as answer]
So overriding in this case is done via 
```
        <SolidColorBrush Color="#F9F8BD" x:Key="TextControlBackgroundPointerOver"/>
        <SolidColorBrush Color="#F9F8BD" x:Key="TextControlBackgroundFocused"/>
        <SolidColorBrush Color="#F9F8BD" x:Key="TextControlBackground"/>
```
Kudos to [Hakob Chalikyan](https://github.com/HakobChalikyan)
